packages/node_modules/@webex/plugin-meetings/src/roap/util.js

```diff
@@ -0,0 +1,89 @@
+import PeerConnectionManager from '../peer-connection-manager';
+import {_ANSWER_, ERROR, CONFLICT, ROAP, SDP} from '../constants';
+
+const RoapUtil = {};
+
+RoapUtil.shouldHandleMedia = (meeting) => {
+  const mediaOffer = meeting.mediaPeerConnection && meeting.mediaPeerConnection.signalingState === SDP.HAVE_LOCAL_OFFER;
+  const screenOffer = meeting.screenPeerConnection && meeting.screenPeerConnection.signalingState === SDP.HAVE_LOCAL_OFFER;
+  if (mediaOffer || screenOffer) {
+    return false;
+  }
+  return true;
+};
+
+RoapUtil.handleError = (pc) => PeerConnectionManager.rollBackLocalDescription({mediaPeerConnection: pc})
+  .then(() => Promise.resolve(true))
+  .catch((err) => Promise.reject(err));
```

Comment:
If we aren't doing any logging or actions, you don't need the `.then` and `.catch` blocks. I guess if you explicitly want to return a `true` boolean on success then you can keep `.then(() => true)`

Reply:
yeah explicitly want to return true